src/View/Helper/FormHelper.php

```diff
@@ -1707,13 +1707,21 @@ public function postLink($title, $url = null, array $options = [])
         unset($options['block']);
 
         $url = '#';
-        $onClick = 'document.' . $formName . '.submit();';
-        if ($confirmMessage) {
-            $options['onclick'] = $this->_confirm($confirmMessage, $onClick, '', $options);
+
+        if(!empty($options['class'])) {
```

Comment:
Yeah ok. In #7511 I mentioned it already that I also didn't update the TestCases yet, as it should be a quick solution to help the discussion in the original issue. So I didn't really look at the test results. But I will surely fix it. :wink: